modules/ROOT/pages/lb-mapping-rules.adoc

```diff
@@ -8,128 +8,262 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies", title="Runtime Fabric"]
 
-The load balancer configuration is defined by a list of mapping rules that describe how an input URL should be translated into calls to different CloudHub apps.
-Mapping rules are attributes of the load balancer's SSL endpoint.
-When you create a mapping rule, you need to specify a certificate CN. Omitting the `[certificateName]` parameter adds the mappings to the default endpoint.
 
-When you create a simple matching rule, one input address is matched to the defined output: the endpoint of one of your applications.
-Instead of using literal matchings, you can also use a pattern to match a variable, such as input text to an endpoint.
+The CloudHub dedicated load balancer (DLB) routes requests from clients to Mule apps deployed within the VPC.
+_Mapping rules_ enable you to rename requests to the DLB (input URL) to a different Mule application name and domain.
```

Comment:
"Rename" is confusing, it implies URL rewrites happening, which may not be the case. I'd say something along the lines of "enables you to forward requests to the DLB to a different..."